src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,694 @@
+---
+title: 'Tutorial: Manage users and groups with the SCIM API (New Relic One user model)'
+tags:
+  - Accounts
+  - Accounts and billing
+  - Automated user management
+  - New Relic One user management
+metaDescription: 'A New Relic user management tutorial: Automated User Management with the SCIM API.'
+---
+
+This tutorial will walk you through some common procedures for managing users on the New Relic One user model using the SCIM API. The SCIM API allows you to view, create, update, and delete users and groups programmatically, outside of the [User management UI](#TODO).
+
+## Requirements [#requirements]
+
+Some notes on who this tutorial is for and requirements:
+
+* This tutorial is for managing users on our New Relic One user model.
+* This tutorial is for managing users without a supported identity provider. Refer to one of the specific integration guides to manage users with a supported identity provider: [Okta](/docs/accounts/accounts/automated-user-management/okta-scimsso-application-configuration/), [Azure AD](/docs/accounts/accounts/automated-user-management/azure-ad-scimsso-application-configuration/), [OneLogin](/docs/accounts/accounts/automated-user-management/azure-ad-scimsso-application-configuration/)
+* Managing users and groups with the SCIM API requires Enterprise edition. For details, see [user management requirements](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles#requirements).
+* Note that managing _access grants_ is not possible with the SCIM API, you must use the **Organization and access** UI to create grants. For details, see [user management tutorial](#TODO).
+* [Role](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#standard-roles) requirements:
+    * Generating the SCIM API token requires the **Authentication domain manager** role.
+    * Using the **Organization and access** UI to manage group access requires **Authentication domain manager** and **Organization manager** roles.
+
+## Overview [#overview]
+
+Before using this tutorial, we first recommend reading these docs:
+
+* [Where to find user management features in the UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where)
+* [Important user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)
+* [SCIM API reference](/docs/accounts/accounts/automated-user-management/scim-support-automated-user-management/)
+* SCIM 2.0 RFC documents from the Internet Engineering Task Force: [RFC 7642](https://tools.ietf.org/html/rfc7642), [RFC 7643](https://tools.ietf.org/html/rfc7643), [RFC 7644](https://tools.ietf.org/html/rfc7644)
```

Comment:
This seems like too much to read. I think I could get pretty far in building out a script without reading these, and the specific sections below are much better. Maybe move those up here? 

Alternatively, you could have another section that's called "Reference materials" or something.